torchvision/transforms/transforms.py

```diff
@@ -1707,6 +1713,7 @@ class RandomInvert(torch.nn.Module):
     The image can be a PIL Image or a torch Tensor, in which case it is expected
     to have [..., H, W] shape, where ... means an arbitrary number of leading
     dimensions.
+    For PIL images, only mode "L" and "RGB" is supported.
```

Comment:
I think that's true for both PIL and Tensor.

Reply:
Let me check if my understanding is correct: it should be that all methods that use _lut() supports only L and RGB for PIL, uint8 for tensor?

Reply:
Both require that they are L or RGB. See the entering invert() functional method:

https://github.com/pytorch/vision/blob/90645ccd0e774ad76200245e32222a23d09f2312/torchvision/transforms/functional.py#L1177-L1178

Reply:
Should I specify the related docs (invert, solarize, etc.) like, for example:

"""Invert the colors of an RGB/grayscale image.

    Args:
        img (PIL Image or Tensor): L or RGB image to have its colors inverted.
            If img is a Tensor, it is expected to be in [..., 1 or 3, H, W] format,
            where ... means it can have an arbitrary number of leading dimensions.

    Returns:
        PIL Image or Tensor: Color inverted image.
"""